nemo_skills/pipeline/utils/mounts.py

```diff
@@ -56,6 +56,18 @@ def check_if_mounted(cluster_config, path_to_check):
         raise ValueError(f"The path '{path_to_check}' is not mounted. Check cluster config.")
 
 
+def _resolve_path_placeholders(path: str) -> str:
+    """Resolve environment variable placeholders in a path."""
+    if path is not None and "${" in path and "}" in path:
```

Comment:
This condition might have edge cases in case the path actually contains `${` as an escaped string, but perhaps it is very unlikely and we can let this go.